overlord/overlord.go

```diff
@@ -347,6 +346,29 @@ func (o *Overlord) Settle(timeout time.Duration) error {
 	return nil
 }
 
+// Settle runs first a state engine Ensure and then wait for
+// activities to settle. That's done by waiting for all managers
```

Comment:
I think "all managers activities" is a little off, maybe "all managers' activities"?